pkg/server/handler/user.go

```diff
@@ -139,6 +139,32 @@ func (payload *userUpdatePayload) Validate() skyerr.Error {
 	return nil
 }
 
+// UserUpdateHandler update an UserInfo with the supplied information.
+//
+// UserUpdateHandler allow user to update his attributes with provided ID.
```

Comment:
“UserUpdateHandler updates the following user attributes of the provided User ID.